packages/api/test/e2e/getMetadata.spec.ts

```diff
@@ -0,0 +1,49 @@
+// Copyright 2019-2021 Centrality Investments Limited
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {getMetadata} from "../../src/util/getMetadata";
+import {TypeRegistry} from "@polkadot/types/create";
+import {MetadataVersioned} from "@polkadot/metadata/MetadataVersioned";
+
+describe.skip('getMetadata()',  () => {
```

Comment:
did you mean to unskip this?

Reply:
yes..  I am trying to run this as a part of e2e test... looking at this currently